frameworks/elastic/src/main/dist/svc.yml

```diff
@@ -51,13 +49,11 @@ pods:
           size: {{MASTER_NODE_DISK}}
         cmd: |
           set -e
-
```

Comment:
Was there a reason for removing these newlines? (in all the command blocks).

Reply:
These lines have nothing to do with the basetech update. They are here here to aid in readability in the multi-line command that is executed by the scheduler. Please restore the whitespace.

Reply:
restored the whitespace.